docker/docker-compose.yml

```diff
@@ -68,6 +68,34 @@ services:
       - 'traefik.http.middlewares.strip_auth_prefix.stripprefix.forceSlash=true'
       - 'traefik.http.routers.auth.middlewares=strip_auth_prefix'
 
+  search:
+    container_name: 'search'
+    build:
+      context: ../src/api/search
+      dockerfile: Dockerfile
+    environment:
+      - ELASTIC_APM_SERVICE_NAME=search
+      - SEARCH_PORT
+
```

Comment:
Remove this

Reply:
@humphd Can you please clarify what should be removed?

Reply:
Remove the blank line

Reply:
Done